### PR TITLE
Feature/framework cloned

### PR DESCRIPTION
**Pull request by k9845**
Addresses
 - https://github.com/the-deep/server/issues/1308
 - https://github.com/the-deep/server/issues/1307

## Changes
 - Added cloned_from in analysis_framework
 - Remove `visible_projects` from framework query.

Mention related users here if any.

## This PR doesn't introduce any:

- [ ] temporary files, auto-generated files or secret keys
- [ ] n+1 queries
- [ ] flake8 issues
- [ ] `print`
- [ ] typos
- [ ] unwanted comments

## This PR contains valid:

- [ ] tests
- [ ] permission checks (tests here too)
- [ ] translations

### CODE REVIEW

File: schema.graphql
Comment (thenav56) [inline schema.graphql:1]: @AdityaKhatri Check if this is as needed.

File: apps/analysis_framework/tests/test_schemas.py [hunk 1]
```diff
@@ -207,21 +202,7 @@ def _query_check(**kwargs):
...
-        # lets add member to the private_project
-        private_project.add_member(user)
-        response = _query_check()['data']['analysisFramework']
-        self.assertEqual(len(response['visibleProjects']), 2, response)
-        self.assertListIds(
-            response['visibleProjects'],
-            [project, private_project],
-            response
-        )
+
```
Comment (thenav56) [inline apps/analysis_framework/tests/test_schemas.py:208]: Let's add a test for clone_from as well. I should be in the REST test cases.

**Review by thenav56 — CHANGES_REQUESTED** [review]
Looks good.

File: apps/analysis_framework/tests/test_schemas.py (hunk 1 above)
Comment (k9845) [reply apps/analysis_framework/tests/test_schemas.py:208]: @thenav56 Are you referring to mutation testcase or schema test case here?

Comment (k9845) [reply apps/analysis_framework/tests/test_schemas.py:208]: Added testcase in schema.

**Pull request merged**